### ISSUE TITLE
TrapDoor crash

### ISSUE THREAD

**Issue by Pub4Game** [Nukkit/Nukkit#253]
![image](https://cloud.githubusercontent.com/assets/11244874/12037529/9e9808ac-ae60-11e5-9d57-6135162f6084.png)

**Comment by xpyctum** [comment]
```java
int[] faces = {
                    0,
                    1,
                    2,
                    3,
                    4, //add this
                    5, //and this
            };
```
Maybe it help.

**Comment by Pub4Game** [comment]
fix #254
